extensions/resource-deployment/src/services/arcService.ts

```diff
@@ -1,17 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Just delete this file

Reply:
hmmm. In my enlistment, I do not see this file.